x-pack/platform/plugins/shared/streams_app/test/scout/ui/tests/enable_wired_streams_flow.spec.ts

```diff
@@ -0,0 +1,257 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { expect } from '@kbn/scout';
+import { test } from '../fixtures';
+
+// Common test IDs
+const WIRED_SWITCH = 'streamsWiredSwitch';
+const SETTINGS_FLYOUT = 'streamsSettingsFlyout';
```

Comment:
That id doesn't exist as a `data-test-subj` and because we check `toBeHidden()` it gives us false positive result